app/app.rb

```diff
@@ -293,14 +294,27 @@ def process_form_data(data, git)
 					file.relative_path.to_s
 				elsif data.keys.include?('___checkbox_unchecked')
 					data.keys.include?('___checkbox_checked')
+				elsif data.keys.include?("___datetime_date")
+					datetime = data.values.join(" ")
```

Comment:
We are defining the order, since we made the templates. But maybe it is best to do what I had before and interpolate the values explicitly.